test/development/basic/__snapshots__/next-rs-api.test.ts.snap

```diff
@@ -1,58 +1,1374 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Fixing the gitattributes for these files in #70421 so that this gets collapsed in code reviews.